addons/nametags/XEH_postInit.sqf

```diff
@@ -23,5 +23,24 @@ GVAR(ShowNamesTime) = -10;
 [29, [false, false, false]], false] call cba_fnc_addKeybind; //LeftControl Key
 
 
-// Draw handle
-addMissionEventHandler ["Draw3D", {_this call FUNC(onDraw3d);}];
+// Draw handle on start if set
+if (GVAR(showPlayerNames) > 0) then {
+    GVAR(drawHandler) = addMissionEventHandler ["Draw3D", {_this call FUNC(onDraw3d);}];
+};
+
+// Set the EH which waits for a setting to be changed, so that the effect is shown immediately
+if (!GVAR(showPlayerNamesForce)) then {
```

Comment:
That can only be changed in module/userconfig though.

The idea why it isn't inside `SettingChanged` is so that EH doesn't get installed at all, as if `showPlayerNamesForce` is enabled clients can't set it anyways.

What would you recommend for lines 27-29?